holidays/countries/guinea_bissau.py

```diff
@@ -0,0 +1,133 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.calendars import _CustomIslamicHolidays
+from holidays.calendars.gregorian import MAR, APR, MAY, JUN, JUL, AUG, SEP, OCT
+from holidays.groups import ChristianHolidays, InternationalHolidays, IslamicHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class GuineaBissau(HolidayBase, ChristianHolidays, InternationalHolidays, IslamicHolidays):
+    """Guinea-Bissau holidays.
+
+    References:
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_Guinea-Bissau>
+    """
+
+    country = "GW"
+    default_language = "pt_GW"
+    # %s (estimated).
+    estimated_label = tr("%s (prevista)")
+    # %s (observed).
+    observed_label = tr("%s (ponte)")
+    # %s (estimated, observed).
+    observed_estimated_label = tr("%s (prevista, ponte)")
+    supported_languages = ("en_US", "pt_GW")
+    start_year = 1975
+
+    def __init__(self, *args, islamic_show_estimated: bool = True, **kwargs):
+        """
+        Args:
+            islamic_show_estimated:
+                Whether to add "estimated" label to Islamic holidays name
+                if holiday date is estimated.
+        """
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        IslamicHolidays.__init__(
+            self, cls=GuineaIslamicHolidays, show_estimated=islamic_show_estimated
+        )
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_new_years_day(tr("Ano Novo"))
+
+        # National Heroes' Day.
+        self._add_holiday_jan_20(tr("Dia dos Heróis Nacionais"))
+
+        # International Women's Day.
+        self._add_womens_day(tr("Dia Internacional da Mulher"))
+
+        # Good Friday.
+        self._add_good_friday(tr("Sexta-feira Santa"))
+
+        # Easter Sunday.
+        self._add_easter_sunday(tr("Páscoa"))
+
+        # Labor Day.
```

Comment:
```suggestion
        # Worker's Day.
```
As traditional for Portuguese-speaking countries. :)